ArdalisSpecificationEF/tests/Ardalis.Specification.EF.IntegrationTests/RepositoryOfT_GetBySpec.cs

```diff
@@ -77,5 +77,17 @@ public async Task ReturnsCompanyWithStoresAndProducts_GivenCompanyByIdIncludeSto
             result.Stores.Count.Should().BeGreaterThan(49);
             result.Stores.Select(x => x.Products).Count().Should().BeGreaterThan(1);
         }
+
+        [Fact]
+        public async Task ReturnsUntrackedCompany_GivenCompanyByIdAsUntrackedSpec()
+        {
+            dbContext.ChangeTracker.Clear();
```

Comment:
We're using a shared database for the tests, all tests working with the same dbContext. In the case of some other test relying somehow on the tracker, clearing the tracker might lead to inconsistent results.

For now, we're good, but let's keep that in mind.